### ISSUE TITLE
Should use UTC by default

### ISSUE THREAD

**Issue by beorn7** [prometheus/alertmanager#363]
Apparently, all the times are displayed in the local timezone of the browser. While that is a nice option to have, the default display should use UTC (consistent with other Prometheus components).

Due to so-called "daylight saving time" in some local timezones, it is also impossible to schedule certain silence ranges if local time is used in the silence form.

**Comment by mxinden** [comment]
@beorn7 This is fixed with the new UI rewrite which will be released with v0.7. I will close here if you don't mind?

**Comment by beorn7** [comment]
We usually close issues once the fix is merged into master.

**Comment by mxinden** [comment]
It was merged with #789. Sorry, I forgot to link this issue in the PR.

**Comment by stuartnelson3** [comment]
Closed by #789
